packages/gen-typescript-declarations/src/ts-ast/declarations.ts

```diff
@@ -401,10 +401,10 @@ export class ConstValue {
 }
 
 /**
- * An identifier that is imported or exported, possibly with a different name.
+ * An identifier that is imported, possibly with a different name.
  */
-export interface ImportExportIdentifier {
-  identifier: string;
+export interface ImportSpecifier {
+  identifier: string|typeof AllIdentifiers;
```

Comment:
cool!

You can also do `type AllIdentifiers = typeof AllIdentifiers`

Reply:
Good idea, done.